src/quantize.rs

```diff
@@ -17,6 +17,8 @@ use std::convert::Into;
 use std::mem;
 use std::ops::AddAssign;
 
+pub const QSCALE: usize = 3;
```

Comment:
Can you please define this constant with a comment? It's a bit far removed from where it's actually used.

Reply:
Found out that this is actually a duplicate.
I'll use [this](https://github.com/xiph/rav1e/blob/master/src/rate.rs#L50) instead.